modules/account-lib/src/coin/cspr/transactionBuilder.ts

```diff
@@ -1,36 +1,68 @@
 import BigNumber from 'bignumber.js';
 import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
-import { BaseTransactionBuilder } from '../baseCoin';
-import { BuildTransactionError, NotImplementedError } from '../baseCoin/errors';
+import { DeployUtil, PublicKey } from 'casper-client-sdk';
+import { parseInt } from 'lodash';
+import { BaseTransactionBuilder, TransactionType } from '../baseCoin';
+import { BuildTransactionError, NotImplementedError, SigningError } from '../baseCoin/errors';
 import { BaseAddress, BaseFee, BaseKey } from '../baseCoin/iface';
 import { Transaction } from './transaction';
 import { KeyPair } from './keyPair';
+import { GasFee, CasperModuleBytesTransaction, CasperTransferTransaction, SignatureData } from './ifaces';
+import { isValidPublicKey } from './utils';
+import { SECP256K1_PREFIX, CHAIN_NAME, MAXIMUM_DURATION } from './constants';
 
 export const DEFAULT_M = 3;
+export const DEFAULT_N = 2;
 export abstract class TransactionBuilder extends BaseTransactionBuilder {
   private _source: BaseAddress;
-  private _fee: BaseFee;
+  protected _fee: GasFee;
   private _transaction: Transaction;
+  protected _session: CasperTransferTransaction | CasperModuleBytesTransaction;
+  protected _duration: number;
+  protected _multiSignerKeyPairs: KeyPair[];
+  protected _signatures: SignatureData[];
 
   constructor(_coinConfig: Readonly<CoinConfig>) {
     super(_coinConfig);
     this.transaction = new Transaction(_coinConfig);
+    this._multiSignerKeyPairs = [];
+    this._signatures = [];
   }
 
   // region Base Builder
   /** @inheritdoc */
   protected async buildImplementation(): Promise<Transaction> {
-    throw new NotImplementedError('buildImplementation not implemented');
+    const deployParams = this.getDeployParams();
+    const session = this.getSession();
+
+    // @ts-ignore
+    const payment = DeployUtil.standardPayment(parseInt(this._fee.gasLimit));
```

Comment:
Why is the the `@ts-ignore` comment needed here?
Should we pass a arg type?

Reply:
The @ts-ignore is required here because the SDK method takes a BigInt implementation which comes from an external library. 
To avoid including this library into our project, we added this tag so we can pass an integer instead.

Reply:
Gotcha. Can you add a that in a comment for clarification?